.eslintrc.js

```diff
@@ -41,6 +41,8 @@ module.exports = {
         bracketSpacing: true,
       },
     ],
+
+    'react/react-in-jsx-scope': 'off',
```

Comment:
People using the React ESLint plugin get errors if they forget to import `react` in a file with JSX. But for the sake of argument I turned it off on our repo and updated an integration test to rely on the automatic importing.

Reply:
Gotcha — if this will be the encouraged default, I think we should remove the import from all of our integration tests.

Reply:
Just to clarify, I don't think we should encourage disabling this ESLint rule. The main reason to auto-import React is just to avoid instances of users forgetting to import it and getting blank white snapshots. Explicitly importing it is still the standard in the React dev world.

I'll remove this change from the global ESLint config and just disable it in that one file. I just wanted to validate that auto-importing React works.